README.md

```diff
@@ -20,13 +20,13 @@ Prerequisites for building Fluss:
 
 - Unix-like environment (we use Linux, Mac OS X, Cygwin, WSL)
 - Git
-- Maven (we require version >= 3.8.6)
+- Maven (we require version >= 3.8.6). A compatible Maven Wrapper (`mvnw`) is included in this project.
```

Comment:
We can revert this if we explain mvnw in the following, as this is not a prerequisite to building Fluss.